src/std/box-niche.md

```diff
@@ -18,15 +18,13 @@ allows the compiler to optimize the memory layout:
 
 ```bob
  Stack                           Heap
-.- - - - - - - - - - - -.     .- - - - - - - - - - - - - - - - - - - - - - -.
-:                       :     :                                             :
-:    list               :     :                                             :
-:   +----+----------+   :     :    +----+----------+    +----+----------+   :
-:   | .0 | ".1/Tag" |   :     :    | .0 | ".1/Tag" |    | .0 | ".1/Tag" |   :
-:   +----+----------+   :     :    +----+----------+    +----+----------+   :
-:   | 1  |    o-----+---+-----+--->| 1  |    o-----+--->| 2  |  null    |   :
-:   +----+----------+   :     :    +----+----------+    +----+----------+   :
-:                       :     :                                             :
-:                       :     :                                             :
-`- - - - - - - - - - - -'     '- - - - - - - - - - - - - - - - - - - - - - -'
+.- - - - - - - - - - - - -.     .- - - - - - - - - - - - - - - - - - - - - - -.
+:                         :     :                                             :
+:    list                 :     :                                             :
+:   +----+----+           :     :    +----+----+    +----+----+               :
+:   | 1  | o--+-----------+-----+--->| 2  | o--+--->| // | // |               :
```

Comment:
I'd still leave a `null` in the second half of the last box, to make it clear that a null pointer is how the compiler notices that the variant is `Nil`.

Reply:
Thanks, that's a nice idea

```suggestion
:   | 1  | o--+-----------+-----+--->| 2  | o--+--->| // | null |             :
```